recompose-composer/src/main/kotlin/recompose/composer/writer/KotlinWriter.kt

```diff
@@ -224,6 +225,21 @@ internal class KotlinWriter {
         }
     }
 
+    private fun writeKeyboardType(value: ParameterValue.KeyboardTypeValue) {
+        writer.continueLine("KeyboardType")
+        writer.continueLine(".")
+        val inputType = when (value.inputType) {
+            InputType.Text -> "Text"
+            InputType.Number -> "Number"
+            InputType.Phone -> "Phone"
+            InputType.Uri -> "Uri"
+            InputType.Email -> "Email"
+            InputType.Password -> "Password"
+            InputType.NumberPassword -> "NumberPassword"
+        }
+        writer.continueLine(inputType)
+    }
+
```

Comment:
Should I use Reflection to get the name of the object here instead on `when` statement?

Reply:
No, I think the `when` statement is great here. :)